scst-store/using-encryption-and-connection.hbs.md

```diff
@@ -1,161 +1,22 @@
 # Configure target endpoint and certificate
 
-The connection to the Store requires TLS encryption and the configuration depends on the kind of installation. Use the following instructions to set up the TLS connection according to the your setup:
+The connection to the Supply Chain Security Tools - Store requires TLS encryption and the configuration depends on the kind of installation.
 
-- [With `Ingress`](#ingress)
-- [Without `Ingress`](#no-ingress)
-    - [`LoadBalancer`](#use-lb)
-    - [Port forwarding](#config-pf)
-    - [`NodePort`](#use-np)
-    
-VMware recommended connection methods based on Tanzu Application Platform setup:
+For a production environment, VMware recommends that Supply Chain Security Tools - Store is installed with ingress enabled. The following instructions will help set up the TLS connection assuming that you've deployed with ingress enabled.
 
-* Single or multi-cluster with Contour = `Ingress`
-* Single cluster without Contour and with `LoadBalancer` support = `LoadBalancer`
-* Single cluster without Contour and without `LoadBalancer` = Port forwarding
-* Single cluster without Contour, without `LoadBalancer` and user does not have port forwarding access = `NodePort`
-* Multi-cluster without Contour = Not supported
+## Using `Ingress`
 
-For a production environment, VMware recommends that the Store is installed with ingress enabled. 
+When using an [Ingress setup](ingress-multicluster.md), the Supply Chain Security Tools - Store creates a specific TLS Certificate for HTTPS communications under the `metadata-store` namespace.
```

Comment:
Ingress doc is now in its own page now. You'll have to pull upstream to see it. `ingress.hbs.md`.